### PR TITLE
changelog 12/11

### PR DESCRIPTION
**Pull request by Ankush-lastmile**
changelog 12/11

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
 
+## Python Version 1.1.5 (2023-12-11)
+
+## NPM Version 1.0.7 (2023-12-11)
+
+Full Changelog:
+
+### Features
+
+- **python-sdk** Evaluation harness with AIConfig. Supports text input/output evaluation with native AIConfig Interface as well as an integration with Promptfoo. (Link to ReadME)
```
Comment (rossdanlm) [inline CHANGELOG.md:11]:
```suggestion
- **python-sdk** Evaluation harness for AIConfig. Supports text input/output evaluation with native AIConfig Interface ([tutorial](https://github.com/lastmile-ai/aiconfig/blob/9669b9e5614d5b1b0603ddf3ec61556cd79bfc14/python/src/aiconfig/eval/examples/travel/travel_eval.ipynb)) as well as an integration with Promptfoo ([tutorial](https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/eval/README.md#usage-guide---promptfoo-integration)). See [README](https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/eval/README.md) for details
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## Python Version 1.1.5 (2023-12-11)
```
Comment (rossdanlm) [inline CHANGELOG.md:3]: 1.1.5

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+- **typescript-sdk:** Support for PaLM Text as a core model parser ([222bf6e](https://github.com/lastmile-ai/aiconfig/pull/348/commits/222bf6edf5c615b3b9b5f58b7fd5c829b2884d1b))
+- - Link to video demo
+
+### Bug Fixes
+
+- **python-extension** HuggingFace Transformers Extension: Fixed bug where we're not properly appending outputs for multiple return sequences (https://github.com/lastmile-ai/aiconfig/pull/411)
+
+### Extensions
+
+- **python-extension:** Extension for LLama-Guard using pytorch ([222bf6e](https://github.com/lastmile-ai/aiconfig/pull/348/commits/222bf6edf5c615b3b9b5f58b7fd5c829b2884d1b))
```
Comment (rossdanlm) [inline CHANGELOG.md:22]: Also include not defaulting to gpt2 for model name

**Review by rossdanlm — APPROVED** [review]
lgtm!

**Comment by Ankush-lastmile** [comment]
Publishing this changelog diff. Can iterate on top for any other changes

**Pull request merged**